tests/unit/utils/test_usage_tracking.py

```diff
@@ -103,7 +103,7 @@ def test_device_id_persisted():
     # preserve the existing dev id so that we dont keep regenerating
     # random device ids for this test machine
     old_devid_path = devid_path.parent / (DEVICE_ID_FILE_NAME + ".old")
-    shutil.move(devid_path, old_devid_path)
+    shutil.move(str(devid_path), old_devid_path)
```

Comment:
This was necessary to pass mypy pre-commit hook.

Reply:
@lionsardesai this is fine right?

Reply:
@pd-t can you check if your version of mypy is the same as the one in precommit config?
It should be 
```
mypy==0.931
mypy-extensions==0.4.3
```

Reply:
![image](https://user-images.githubusercontent.com/67829287/210530606-db57e3da-d1db-4f56-8b30-16497a3b0f71.png)

![image](https://user-images.githubusercontent.com/67829287/210530665-4ef37140-ab5d-47ff-9486-ebc3a9e409bb.png)

Looks quite fine to me.

Reply:
ok lets leave this as is. thanks for checking for us!